core/src/stmt.rs

```diff
@@ -24,3 +24,55 @@ pub trait MongoStatement: Debug {
             .map_or(Err(Error::ColIndexOutOfBounds(col_index)), Ok)
     }
 }
+
+#[derive(Debug)]
+pub struct EmptyStatement {
+    pub resultset_metadata: Vec<MongoColMetadata>,
+}
+
+impl MongoStatement for EmptyStatement {
```

Comment:
I like this! Can you use it in collections.rs for all_schemas too?
```
    pub fn all_schemas() -> EmptyStatement {
        EmptyStatement {
            resultset_metadata: &COLLECTIONS_METADATA,
        }
    }
```

Reply:
Great idea!